app/utils/aws.ts

```
@@ -0,0 +1,432 @@
+        let text = "";
+        if (parsed.generation) {
+          text = parsed.generation;
+        } else if (parsed.output) {
+          text = parsed.output;
+        } else if (typeof parsed === "string") {
+          text = parsed;
+        }
+
+        if (text) {
+          // Clean up any control characters or invalid JSON characters
+          text = text.replace(/[\x00-\x1F\x7F-\x9F]/g, "");
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)